src/middlewares/AuthValidator.js

```diff
@@ -0,0 +1,68 @@
+
+/**
+  * @param {object} request - request object
```

Comment:
Ensure your JSDoc adheres to our team convention as discussed this morning. Also, next is not a type, it should be a function ie. `@param {Function} next - calls next middleware`